src/seed/index.ts

```diff
@@ -0,0 +1,63 @@
+import { Payload } from 'payload'
+
+async function all_collections_empty(payload: Payload): Promise<boolean> {
+  const count_users = await payload.count({ collection: "users" });
+  const count_events = await payload.count({collection: "events"});
+  const count_media = await payload.count({collection: "media"});
+  return count_events.totalDocs == 0 && count_events.totalDocs == 0 && count_media.totalDocs == 0;
+}
+
+export const seed = async (payload: Payload): Promise<void> => {
+  if (!(await all_collections_empty(payload))) {
+    return ;
+  }
+  
+  payload.logger.info('Seeding sample data...')
+
+  await payload.create({
```

Comment:
I think it would be helpful to log that these two users were created and also show their credentials. That way we won't have to look into the code to find them :)

Reply:
they are logged now